### ISSUE TITLE
If you zoom out enough, show a map of all agencies

### ISSUE THREAD

**Issue by hathix** [trynmaps/metrics-mvp#440]
The map on the homescreen is currently restrained to a certain max zoom distance (good). But once we start supporting multiple transit agencies, it'd be really cool to remove that zoom cap and let people zoom out to a country-level view -- and then see all the different agencies on the map. This way we could advertise all the different agencies we have and enable better cross-agency comparison.

**Comment by youngj** [comment]
I don't think it has actually been decided if we want to have a centralized web app with data from all cities in one place. See https://github.com/trynmaps/metrics-mvp/issues/456
